Library/Formula/python.rb

```diff
@@ -115,7 +126,7 @@ def install
     # superenv handles that cc finds includes/libs!
     inreplace "setup.py" do |s|
       s.gsub! "do_readline = self.compiler.find_library_file(lib_dirs, 'readline')",
-              "do_readline = '#{Formula["readline"].opt_lib}/libhistory.dylib'"
+              "do_readline = '#{Formula["readline"].opt_lib}/libhistory.so'"
```

Comment:
No, but Python fails to detect whether readline is linked against curses or not, which can cause the _curses module to misbuild.